libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -164,28 +177,276 @@ def draw_mermaid_png(
     Raises:
         ValueError: If an invalid draw method is provided.
     """
-    if draw_method == MermaidDrawMethod.PYPPETEER:
+    if draw_method == "api":
+        draw_method_ = MermaidDrawMethod.API
+    elif draw_method == "playwright":
+        draw_method_ = MermaidDrawMethod.PLAYWRIGHT
+    elif draw_method == "pyppeteer":
+        draw_method_ = MermaidDrawMethod.PYPPETEER
```

Comment:
how does this one work in sync?

Reply:
Currently, kicks off an event loop even if one exists already (so requires usage of nest_asyncio). We can kick it off on a thread to fix this up. (Don't think it offers a sync API.)